docs/export_themes.js

```diff
@@ -69,6 +70,7 @@ themeConfigOverrrides.set('zash.omp.json', newThemeConfig(40, 40));
     let poshCommand = `oh-my-posh --config=${configPath} --shell shell --export-png`;
     poshCommand += ` --rprompt-offset=${config.rpromptOffset}`;
     poshCommand += ` --cursor-padding=${config.cursorPadding}`;
+    poshCommand += ` --bg-color=${config.cursorPadding}`;
```

Comment:
I think here is a bug. Should it not be `--bg-color=${config.bgColor}`?

Reply:
That's been fixed in a [later PR](https://github.com/JanDeDobbeleer/oh-my-posh/commit/c6a3dd4ee8e4ffa323a116fe7bc9be8e58258985).